### DISCUSSION TITLE
Strange behaviour for elasticSearchExchangeStore

### DISCUSSION THREAD

**Discussion by Lucamadio** [membrane/api-gateway#1285] [General]
Hi guys,
i have some question related to elasticSearchExchangeStore.

1. when i send a request to elastic the body field of the document is base64 encoded. Is there a way to store the body data in UTF-8 String mode?
2. the body field of the response is always empty. This is my configuration in proxies.xml
 
![immagine](https://github.com/user-attachments/assets/5705c2c0-738e-4eb5-913f-e57c6056e3e3)

Is this a bug or i'm making some mistakes?

**Comment by predic8** [comment]
Hi @Lucamadio,
thanks for the question and the detailed description. The base64 encoding is for the case of binary bodies like PDF documents. In case of JSON bodies we can change that to a string.
For the response body I'll come back to you later.
